### PR TITLE
Script to reindex solr using a list of package ids

### PR DESCRIPTION
**Pull request by kentsanggds**
## What

Target solr reindex to problem datasets.

### CODE REVIEW

File: bin/python_scripts/solr_reindex_package_ids.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+import os
+import sys
+import subprocess
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+FILENAME_BASE = "reindex_datasets-package_ids_28-31_07"
```
Comment (kevindew) [inline bin/python_scripts/solr_reindex_package_ids.py:18]: Are you intending to commit this as a file? I notice line 37 seems to open "reindex_datasets-package_ids_28-31_07.txt"

File: bin/python_scripts/solr_reindex_package_ids.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+def reindex_solr():
+    with open("%s.txt" % FILENAME_BASE, "r+") as f:
+        lines = f.readlines()
+        for i, line in enumerate(lines):
+            paster_command = 'paster --plugin=ckan search-index rebuild {} --config /var/ckan/ckan.ini'.format(line.strip())
+
+            logger.info('CKAN reindex %d/%d - Running command: %s', i+1, len(lines), paster_command)
+
+            try:
+                subprocess.call(paster_command, shell=True)
```
Comment (thomasleese) [inline bin/python_scripts/solr_reindex_package_ids.py:45]: I think you'll want to use [`check_call`](https://docs.python.org/2/library/subprocess.html#subprocess.check_call) for it to raise an exception.

File: bin/python_scripts/solr_reindex_package_ids.py (hunk 1 above)
Comment (kentsanggds) [reply bin/python_scripts/solr_reindex_package_ids.py:18]: not really, I'll change it to some generic filename

File: bin/python_scripts/solr_reindex_package_ids.py (hunk 2 above)
Comment (kentsanggds) [reply bin/python_scripts/solr_reindex_package_ids.py:45]: ok, will change

**Pull request merged**